features/upsert.feature

```diff
@@ -41,3 +41,54 @@ Feature: Upsert Cache Time
             """
             """
     Then the HTTP status code should be "400"
+
+  Scenario: Upsert Cache Time resource with empty release_time & collection_id
+    Given the document with "_id" set to "5d41402abc4b2a76b9719d911017c592" does not exist in the "cachetimes" collection
+    When I PUT "/v1/cache-times/5d41402abc4b2a76b9719d911017c592"
+            """
+            {
+                "path": "/my-path",
+                "etag": "test-etag"
+            }
+            """
+   Then the HTTP status code should be "204"
```

Comment:
Very small one, but the indentation of these 3 lines is missing a space.

Reply:
Hawk eyes 😄 - fixed